code/game/objects/items/weapons/storage/storage.dm

```diff
@@ -100,27 +99,16 @@
 			. += G.gift:return_inv()
 
 /obj/item/weapon/storage/proc/show_to(mob/user as mob)
-	if(user.s_active != src)
-		for(var/obj/item/I in src)
-			if(I.on_found(user))
-				return
-	if(user.s_active)
-		user.s_active.hide_from(user)
-	user.client.screen -= boxes
-	user.client.screen -= storage_start
-	user.client.screen -= storage_continue
-	user.client.screen -= storage_end
-	user.client.screen -= closer
-	user.client.screen -= contents
-	user.client.screen += closer
-	user.client.screen += contents
+	for(var/obj/item/I in src)
+		if(I.on_found(user))
+			return
+	
+	user.client.screen |= closer

```

Comment:
Any particular reason for the `|=` operators? It could be wasted work.

Reply:
It originally used `-=` then `+=`, which is, as far as I know, syntactically equivalent.

Reply:
uh. Absolutely not lmao.
`|=` is: "If not found in list, add." It does not remove. It _does_ however scan the entire list, looking to see if it's already in it first.

Reply:
-= then += is the exact same as |= in that it adds it while preventing duplicates. It is functionally the exact same.